admin.html

```diff
@@ -74,9 +78,26 @@ <h2 class="mdl-card__title-text">Create a new DoWhop</h2>
                   <span class="mdl-tooltip" for="doer-description">Enter multiple doers, seperated, by, Commas</span>
                 </div>
                 <div class="mdl-card__actions">
-                  <input id="dowhop-image-capture1" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
-                  <input id="dowhop-image-capture2" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
-                  <input id="dowhop-image-capture3" type="file" accept="image/*" capture="camera" onchange="addDoWhopImage(this.files, this)">
+                  <div id="create-outer-div">
```

Comment:
@JohannBillar are you saying to change create-outer-div to something more meaningful? This is the section where users add images. I could change it to add-images-div. Also the individual image icons have divs named create-div-1, create-div-2, create-div-3. I could change these to something like image-1, image-2, image-3. Let me know if you have better suggestions. (And obviously, I will update the css that points to the original names as well)